docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+services:
+    migrate:
+        #image: public.ecr.aws/a4v3h0f0/vikingslog:v1
```

Comment:
Explain in readme the difference for the commented image names and commented lines and how to use them

Reply:
removed because locally we are not using that!

Reply:
I will add logqa.cavepotlab.com